doc/techref/environment_variables.md

```diff
@@ -0,0 +1,47 @@
+# Environmental Variables
```

Comment:
It seems both `environment` and `environmental` variables are used. I fell we should decide for one term.
Looking at this [StackOverflow post](https://stackoverflow.com/questions/70677153/are-they-called-environment-or-environmental-variables) `environment` variable is used more often and considered as "more correct".

```suggestion
# Environment Variables
```